addons/arsenal/functions/fnc_sortStatement_mod.sqf

```diff
@@ -0,0 +1,19 @@
+#include "script_component.hpp"
+/*
+ * Author: SynixeBrett
+ * Statement for the mod sort.
+ *
+ * Arguments:
+ * 0: item config path (CONFIG)
+ *
+ * Return Value:
+ * Mod image to sort by
+ *
+ * Public: No
+*/
+
+private _class = configName _this;
+private _hierarchy = configHierarchy _this;
+private _parent = _hierarchy select ((count _hierarchy) - 1)
+
+(modParams [(getAssetDLCInfo [_class, _parent]) select 5, ["logo"]]) param [0, ""]
```

Comment:
getAssetDLCInfo returns DLC info, I would argue that most modded weapons are not DLC's?

Reply:
Also is this sorting by logo image path? Not by mod name?

Reply:
Many mods set `dlc` config entry properly. That is what BI Arsenal uses as well I believe.

Reply:
Old arsenal scanned CfgPatches, which all mods have.

> Many mods set dlc config entry properly

I'm not sure if getAssetDLCInfo retrieves that config entry.

Reply:
Yeah I'll rewrite this to avoid `getAssetDLCInfo`